### DISCUSSION TITLE
model vs media exchange reaction fluxes

### DISCUSSION THREAD

**Discussion by jmol0917** [micom-dev/micom#113] [Q&A] [answered]
Hi All!

Not sure if I missed this from somewhere in the documentation but I am wondering how are the values in ```solution.fluxes``` calculated.

For example, in a 3-member community ```EX_ddca(e)``` flux is -0.95 for one strain and NaN for the other two, but ```EX_ddca_m``` is -0.43 in the same solution.

Does individual growth rate/biomass has anything to do with this?

Not sure if this is relevant but growth rate for the strain that consumes the metabolite is 3.8 and the other two get to 7.8 and 23.7. Community growth is 7.43.

Thanks!!

**Comment by cdiener** [comment, marked as answer]
Hi, internal fluxes like EX_ddca(e) are given per 1 gDW of the respective taxon, but external fluxes are given per 1 gDW of the total biomass/abundance. In your case, because only one taxon is consuming ddca those are related by `EX_ddca_m = a_i * EX_ddca(e)_i` where `a_i` is the relative abundance of the strain (which seems to be around 0.45 in your case). Basically 1gDW of strain one would consume 0.95 mmol/h of ddca but 1gDW of the microbial consortia only include 0.45 gDW of strain 1 so the total consumption is `0.45 gDW[i]/gDS[total] * -0.95 mmol/(h * gDW[i]) = -0.43 mmol/(h * gDW[total])`.

**Comment by jmol0917** [comment]
Thanks for clarifying @cdiener. Really appreciate the ongoing support and improvement of MICOM.
